.github/workflows/web.yml

```diff
@@ -23,6 +23,12 @@ jobs:
       - name: Install wasm-pack
         uses: jetli/wasm-pack-action@v0.4.0
 
+      - name: Cache WASM build
+        uses: actions/cache@v3
+        with:
+          path: route_info/target
+          key: doesnt-matter-share-everything
```

Comment:
The key can include things like the OS where this is running. We only build on one platform, so irrelevant. We could also hash the lockfile, but that doesn't really make sense either. If a commit changes or adds a dependency to the Rust crate, we can still reuse most of the cache. The rust build tools know how to rebuild only new things.

https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#matching-a-cache-key